src/Microsoft.AspNetCore.Razor.Language/RazorDiagnosticFactory.cs

```diff
@@ -316,6 +316,32 @@ public static RazorDiagnostic CreateTagHelper_InvalidTargetedAttributeName(strin
             return diagnostic;
         }
 
+        internal static readonly RazorDiagnosticDescriptor TagHelper_CodeBlocksNotSupportedInAttributes =
```

Comment:
Hmm, thinking. Should I move these into the semantic diagnostic factory section? Probably should.

Reply:
Probably should. Also, merge conflicts 😢

Reply:
Race you 😉

Reply:
Merging this, when you merge feel free to push the diagnostic #'s of these diagnostics to be after yours ( so you don't need to increment all of yours ).